misc/community-marketing-kit.md

```diff
@@ -0,0 +1,71 @@
+# Guide to promoting Firefox in your language - a community marketing kit
+
+For a community new or well established, it is always exciting to reach the milestone when the localized Firefox is ready for the first time or a major release. It takes months of translation, testing and QA to get to the finish line. However, the work is not done. Now it comes to the challenging part of promoting the product in your language and spreading the word of mouth. Though the guide is not only written for the new Firefox launch, the guide is created in anticipation of a major launch event, is intended to address some of the most frequently questions you asked about promoting your localized Firefox and other Mozilla products.
```

Comment:
I'm quite lost in the sentence starting with "Though…". Can you simplify it a bit, e.g. "The guide is created to support a major launch event, not only for a new Firefox launch, and it addresses…"